validators/http/http.go

```diff
@@ -37,6 +38,7 @@ type Validator struct {
 	failureStatusCodeRanges []*numRange
 	successHeaderRegexp     *regexp.Regexp
 	failureHeaderRegexp     *regexp.Regexp
+	maxLastModifiedDiffSec  time.Duration
```

Comment:
Since it's already parsed time duration, I'd remove Sec from the field name.
```suggestion
	maxLastModifiedDiff  time.Duration
```